src/compiler/checker.ts

```diff
@@ -23628,6 +23628,18 @@ namespace ts {
             }
         }
 
+        function nodeRefersToSuperOrThis(node: Statement): boolean {

```

Comment:
Consider renaming `refersTo` to `references` to make it more consistent with similar function names